model_validation/ciff_sam_results.py

```diff
@@ -107,6 +107,7 @@ def compute_mam_duration(self, strata=DEFAULT_STRATA):
         ['4.0', 'v4.0_wasting_treatment', '2021_09_20_14_45_25'],
         ['4.1', 'v4.1_wasting_treatment', '2021_09_24_16_36_30'],
         ['4.5.2', 'v4.5.2_x_factor', '2021_09_29_12_12_47'],
+        ['4.5.3', 'v4.5.3_x_factor_targeted_exposure', '2021_11_02_20_09_56'],
```

Comment:
THis might be a good place to include more notes on what is different about each version (or link to somewhere else that has documented this)

Reply:
This hardcoded table is a temporary solution... what I really want is a .csv that is automatically generated from the directory structure, and potentially we could add automatically generated links to GitHub pull requests and Jira tickets to the .csv so we'd have a record of what happened in each model update. And the .csv could be displayed in the concept model as well as accessible to the validation code.